src/main/flight/mixer_profile.h

```diff
@@ -0,0 +1,78 @@
+#pragma once
+
+#include "config/parameter_group.h"
+#include "flight/failsafe.h"
+#include "flight/mixer.h"
+#include "flight/servos.h"
+
+#ifndef MAX_MIXER_PROFILE_COUNT
+#define MAX_MIXER_PROFILE_COUNT 2
+#endif
+
+typedef struct mixerConfig_s {
+    int8_t motorDirectionInverted;
+    uint8_t platformType;
+    bool hasFlaps;
+    int16_t appliedMixerPreset;
+    uint8_t outputMode;
```

Comment:
We dropped this from mixerConfig so I think it can be removed

Reply:
removed uint8_t outputMode;